app/scripts/modules/core/cluster/allClustersGroupings.component.html

```diff
@@ -1,8 +1,4 @@
-<div data-scroll-id="clusters-content"
-     waypoint-container
```

Comment:
It doesn't seem to be working in the wizard any more. When I scroll through the wizard, the navigation sections don't highlight.